tools/ci_static_check_reports/build_static_checkers_reports.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+from invoke import Context
+
+sys.path.insert(0, "airbyte-integrations/connectors")
+from tasks import CONFIG_FILE, TOOLS_VERSIONS, _run_task  # noqa
+
```

Comment:
why is the python protected function of an external module imported here?
Will it be able to become a potential issue in the future?

Reply:
I don't think so. But maybe we should changed `_run_task ` to `run_task` as it may be used in different modules in future.